pkgdb2/api/packages.py

```diff
@@ -152,7 +152,7 @@ def api_package_new():
             SESSION.commit()
             output['output'] = 'ok'
             output['messages'] = [message]
-        except pkgdblib.PkgdbException, err:
+        except pkgdblib.exceptions.PkgdbException, err:
```

Comment:
Should we use the same construct as in ``pkgdb2/api/acls.py``? It would allow us to be consistent :)